.vscode/launch.json

```diff
@@ -5,14 +5,25 @@
             "name": "Debug",
             "type": "coreclr",
             "request": "launch",
-            "preLaunchTask": "Start",
+            "preLaunchTask": "Build Server",
```

Comment:
Cc @Krzysztof-Cieslak

Reply:
Start didn't work for me when called through Code's debug / launch. But happy for that to be rolled back if needed.